paths.tex

```diff
@@ -13,48 +15,118 @@
+  of these temporal networks, it may be convenient to define an edge as
+  a 6-tuple with the last component $l$ being a unique label or edge
+  differentiator. This would allow for the following scenario: if author
+  \texttt{A} and \texttt{B} collaborate in window $[t_0, t_f]$ on multiple
+  papers with the same weight, we could differentiate two edges connecting the
+  two authors instead of condensing the two paper collaborations into one edge.
+  In many cases, it may not make sense to do this, but it is valuable to have
+  an umbrella definition that encompasses every type of graph we could create.}
+
+  Informally, if for every directed edge in the graph, there exists a
+  corresponding edge in the opposite direction connecting the same two
+  vertices and labeled with the exact same $t_i, t_f,$ and $w$, we will consider
+  the temporal graph to be undirected.
+\end{defn}
+
+\begin{defn}
+  We will describe a temporal graph as \textbf{unweighted} if every edge in $E$
+  has equal weight $w$.
+\end{defn}
+
+\begin{defn}
+  \textbf{Edge persistence} is the amount of time for which an edge is present.
+  More formally, in a graph $(V,E)$ the persistence of an edge $(u,v,t_1,t_2,
+  w)$ is defined as $t_2 - t_1$ or simply $\delta t$.
+\end{defn}
+
+\begin{defn}
+  An edge $e \in E$ is said to be \textbf{infinitely persistent}
+  if $t_f = \infty$.
+\end{defn}
 
 \begin{defn}
-  An \textbf{edge-varying temporal network} is a tuple $G \in V \times E$ where
-  the set $V$ is the set of vertices, and $E \in V^2 \times T^2$, where $V$ is
-  the set of vertices and $T = [0,\infty)$.  The first vertex represents the
-  A \textbf{vertex-varying temporal network} is a tuple $G \in V \times E$ where
-  the set $V$ is the set of vertices -- each labelled with a start and end time,
-  and $E \in V^2$ , a pair of vertices.
-  When the vertex pair within the edges is ordered, call the graph
-  \textbf{directed}; otherwise call it \textbf{undirected}.
+  A \textbf{instantaneous edge} or \textbf{contact edge} is any edge $e \in E$
+  where $t_0 = t_f$.
 \end{defn}
 
-Specifically we will analyze the effect of these definitions on the coauthorship
-and citation networks.
+\begin{defn}
+  \textbf{Interval networks} or \textbf{windowed networks} shall be defined as
+  any temporal network where for at least one edge $e = (v_i, v_j, t_0, t_f,w)$,
+  $t_0 \neq t_f$.
+\end{defn}
 
+\begin{defn}
+  \textbf{Contact networks} shall be defined as a temporal network where
+  every edge in the network is an \textit{instantaneous edge} or \textit{contact
+  edge}. It is simply a special case of an interval network.
+\end{defn}
 
 \begin{defn}
-  A \textbf{coauthorship network} is an undirected edge-varying temporal network
-  in which each node represents an author, and the existence of an edge represents
-  a collaboration over a period of time.
+  An \textbf{aggregate network} shall be defined as a temporal network where
+  every edge is \textbf{infinitely persistent}.
```

Comment:
I see it as varying over time making it suited as a sub-case. An aggregate network (at least the way defined in this document) allows for varying start times on each of the edges; therefore, if you look at an aggregate network in year `t_n` and then again in `t_{n+1}` it may have changed and therefore varies in time. Edges cannot become "de-activated", but they come into "activation" at different times. Perhaps we may want to choose a different word to refer to this behavior.